estate_account/__manifest__.py

```diff
@@ -0,0 +1,12 @@
+{
+    'name': 'frva-estate-account',
+    'license': 'LGPL-3',
+    'depends': [
+        'base',
```

Comment:
Already a dependency of estate. Usually, we only put 'base' if there is no other dependency.